### ISSUE TITLE
[BUG] Different issues counts between Chrome and FIrefox

### ISSUE THREAD

**Issue by drjoho** [IBMa/equal-access#823]

**Comment by drjoho** [comment]
Here is Firefox counts with version Version 3.1.25

<img width="1072" alt="image" src="https://user-images.githubusercontent.com/64442976/163735081-0a22a778-ec11-4c22-a4f6-a10b0a297b55.png">

**Comment by drjoho** [comment]
Ok I just checked the checker numbers and here are the results for what is currently deployed on chrome and firefox:
Chrome:
![image.png](https://images.zenhubusercontent.com/61081394ffd52d20c363a4cd/24d49793-ecff-47ec-b053-d927e73dfcc1)
Firefox:
![image.png](https://images.zenhubusercontent.com/61081394ffd52d20c363a4cd/12e1fa7b-f88b-4e38-bb32-0ed86ea9dffd)

**Comment by drjoho** [comment]
Note that the numbers for Violations, Needs Review and Recommendations agree in both Chrome and Firefox. What doesn't agree (note the code is exactly the same) is the number of HTML elements on the page which is used to calculate the current status percentage. So I will investigate why I am getting different numbers.

**Comment by drjoho** [comment]
Chrome:
<img width="1911" alt="image" src="https://user-images.githubusercontent.com/64442976/169092604-8bde3415-30a5-4d1b-8ef9-82292144855c.png">

**Comment by drjoho** [comment]
The counts were confusing since we are counting both issues and elements (via xpaths) and then we show the users Element Roles. Element Roles and xpaths (i.e., real elements) are not the same and cannot be mapped 1-1. I did get the numbers of elements (not issues - elements can have more than one issue) that failed and the total number of elements tested to agree in both Chrome and FF however, users would be tempted to try to match them up to the Element Roles and that will never work. So I took those numbers out of the Current Status Card. I am happy to discuss the convoluted details with anyone interested.

**Comment by eirckr** [comment]
Validated
